src/Polly.Core/CircuitBreaker/BrokenCircuitException.cs

```diff
@@ -54,4 +79,13 @@ protected BrokenCircuitException(SerializationInfo info, StreamingContext contex
     }
 #endif
 #pragma warning restore RS0016 // Add public types and members to the declared API
+
+    /// <summary>
+    /// Gets the amount of time before the circuit can become closed, if known.
+    /// </summary>
+    /// <remarks>
+    /// This value is specified when the instance is constructed and may be inaccurate if consumed at a later time.
+    /// Can be <see langword="null"/> if not provided or if the circuit was manually isolated.
+    /// </remarks>
+    public TimeSpan? RetryAfter { get; }
```

Comment:
We should add a test like this one to verify that binary serialization works correctly for .NET Framework usage:

https://github.com/App-vNext/Polly/blob/439cf1515d84c9ecaa1c6379def2cb3c47151df7/test/Polly.Core.Tests/Timeout/TimeoutRejectedExceptionTests.cs#L21-L31

Getting it to pass probably needs similar code to the below adding to ensure the property value is serialized correctly:

https://github.com/App-vNext/Polly/blob/439cf1515d84c9ecaa1c6379def2cb3c47151df7/src/Polly.Core/Timeout/TimeoutRejectedException.cs#L78-L89